apps/sms/style/sms.css

```diff
@@ -37,6 +37,49 @@ gaia-header {
   z-index: 10;
 }
 
+/*
+  toast
+*/
+section[role="toast"] {
```

Comment:
"role" is actually an aria role; it needs to have specific values (see http://www.w3.org/TR/wai-aria/roles) and "toast" is not one of them. Moreover it was a mistake that we used these roles in CSS, we should not do that anymore, please use CSS classes instead.